src/rust/ide/view/src/project.rs

```diff
@@ -129,6 +132,23 @@ impl Model {
         graph_editor_inputs.set_node_expression.emit(&(node_id,Expression::default()));
         graph_editor_inputs.edit_node.emit(&node_id);
     }
+
+    fn show_fullscreen_visualization(&self, node_id:NodeId) {
+        let node = self.graph_editor.model.model.nodes.all.get_cloned_ref(&node_id);
+        if let Some(node) = node {
+            let visualization = node.view.model.visualization.fullscreen_visualization().clone_ref();
+            self.display_object.remove_child(&self.graph_editor);
+            self.display_object.add_child(&visualization);
+            *self.fullscreen_vis.borrow_mut() = Some(visualization);
+        }
+    }
+
+    fn hide_fullscreen_visualization(&self) {
+        if let Some(visualization) = std::mem::take(&mut *self.fullscreen_vis.borrow_mut()) {
+            self.display_object.remove_child(&visualization);
+            self.display_object.add_child(&self.graph_editor);
+        }
+    }
```

Comment:
these should have docs of what hacks they are doing and why do we do the hacks with a link to issue which will provide correct solution (the scene cutting one)

Reply:
Also, does this work in debug-scene? It is implemented in such a module that I'm not sure it works there.

Reply:
IT works in interface.